ext/lib/ipc/open-amp/open-amp/cmake/options.cmake

```diff
@@ -33,36 +33,29 @@ string (TOUPPER ${CMAKE_SYSTEM_PROCESSOR} PROJECT_PROCESSOR_UPPER)
 string (TOLOWER ${MACHINE}                PROJECT_MACHINE)
 string (TOUPPER ${MACHINE}                PROJECT_MACHINE_UPPER)
 
-# Select to build Remote proc master
-option (WITH_REMOTEPROC_MASTER "Build as remoteproc master" OFF)
-if (WITH_REMOTEPROC_MASTER)
-  option (WITH_LINUXREMOTE  "The remote is Linux" ON)
-endif (WITH_REMOTEPROC_MASTER)
-
 # Select which components are in the openamp lib
 option (WITH_PROXY          "Build with proxy(access device controlled by other processor)" ON)
 option (WITH_APPS           "Build with sample applicaitons" OFF)
 option (WITH_PROXY_APPS     "Build with proxy sample applicaitons" OFF)
 if (WITH_APPS)
   if (WITH_PROXY)
     set (WITH_PROXY_APPS ON)
-  elseif ("${PROJECT_SYSTEM}" STREQUAL "linux")
-    set (WITH_PROXY_APPS ON)
   endif (WITH_PROXY)
-  option (WITH_BENCHMARK    "Build benchmark app" OFF)
 endif (WITH_APPS)
-option (WITH_OBSOLETE       "Build obsolete system libs" OFF)
 
-# Set the complication flags
-set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra")
+option (WITH_VIRTIO_MASTER "Build with virtio master enabled" ON)
+option (WITH_VIRTIO_SLAVE "Build with virtio slave enabled" ON)
```

Comment:
Are you planning to have Kconfig symbols to control these options?

Reply:
> Are you planning to have Kconfig symbols to control these options?

Probably, a follow PR.